### PR TITLE
issue:3660200 implement logrotate for ndt plugin

### PR DESCRIPTION
**Pull request by alextabachnik**

### CODE REVIEW

**Review by atolikin94 — COMMENTED** [review]
How does it actually work? Don't see that ndt_logrotate.conf is used somewhere.. Also please add some tests

**Review by egershonNvidia — APPROVED** [review]

File: plugins/UFM_NDT_Plugin/build/config/ndt_logrotate.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/log/ndt.log {
+    compress
+    rotate 10
```
Comment (kedeme) [inline plugins/UFM_NDT_Plugin/build/config/ndt_logrotate.conf:3]: rotate and size should be configurable in the NDT plugin configuration.

**Review by kedeme — CHANGES_REQUESTED** [review]
please fix my comment.

File: plugins/UFM_NDT_Plugin/build/config/ndt_logrotate.conf (hunk 1 above)
Comment (alextabachnik) [reply plugins/UFM_NDT_Plugin/build/config/ndt_logrotate.conf:3]: ? how? it is external mechanism not related to plugin implementation.

**Review by kedeme — APPROVED** [review]
please set a meeting to discuss the long term solution for UFM plugins log rotation.

**Pull request merged**